environment-pipelines/buildspec-apply.yml

```
@@ -16,10 +16,15 @@ phases:
       - echo "Terraform Apply Phase"
```

To make it easier to see where one thing ends and another starts in the output, please could we have some line breaks...
```suggestion
      - echo -e "\nTerraform Apply Phase"
```